R/dsl.R

```diff
@@ -136,7 +137,55 @@ add_package_checks <- function(...,
 #' @importFrom magrittr %>%
 DSL <- R6Class(
   "DSL",
+  public = list(
```

Comment:
Looks like the definition of the `DSL` class is now duplicated.

Reply:
c/p error